Daytime-Nighttime-Interactions/fault-reporting.rst

```diff
@@ -102,7 +103,9 @@ When creating a ticket, make sure to fill in the following fields:
   In the case they are certain of who correct person to follow-up on the fault report, they should be added as a watcher and flagged in a comment. 
   A team will review the fault reports after the night is over and determine the best person or group for follow-up.
 
-- **Labels:** This is not a required field, but may provide more information to the components involved. 
+- **Primary Software Component:** This is not a required field, but may provide more information to the components involved. 
+
+- **Primary Hardware Component:** If you're not sure what hardware was affected or the root cause, use "Other".
```

Comment:
Consider expanding "you're" to "you are".

Reply:
👍